analytical_engine/core/grape_instance.cc

```diff
@@ -676,6 +676,29 @@ bl::result<void> GrapeInstance::clearEdges(const rpc::GSParams& params) {
   return {};
 }
 
+bl::result<rpc::GraphDef> GrapeInstance::getGraphView(
+    const rpc::GSParams& params) {
+#ifdef EXPERIMENTAL_ON
+  std::string view_id = "graph_view_" + generateId();
+  BOOST_LEAF_AUTO(graph_name, params.Get<std::string>(rpc::GRAPH_NAME));
+  BOOST_LEAF_AUTO(view_type, params.Get<std::string>(rpc::VIEW_TYPE));
```

Comment:
You may validate the parameter `VIEW_TYPE` and turn the plain text into enumeration.